instaclustr/resource_azure_peering.go

```diff
@@ -0,0 +1,169 @@
+package instaclustr
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAzureVpcPeering() *schema.Resource {
+	return &schema.Resource{
+		Create: azureresourceVpcPeeringCreate,
+		Read:   azureresourceVpcPeeringRead,
+		Update: resourceVpcPeeringUpdate,
+		Delete: resourceVpcPeeringDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceVpcPeeringStateImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"cdc_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"peer_resource_group": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"peer_subscription_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"peer_vpc_net": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"resource_group": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"subscription_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"peer_subnets": {
+				Type: schema.TypeSet,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Required: true,
+			},
+
+			"v_net": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func azureresourceVpcPeeringCreate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Still need to fix the name of this function, should be `azureResourceVpcPeeringCreate`, note the capital R, same as the other functions in this file